sdk/mediaservices/Microsoft.Azure.Media.LiveVideoAnalytics.Edge/src/Generated/SdkInfo_DirectMethodsforLiveVideoAnalyticsonIoTEdge.cs

```diff
@@ -19,29 +19,18 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphInstanceActivate", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphInstanceDeactivate", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphInstanceDelete", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphInstanceGet", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphInstanceList", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphInstanceSet", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphTopologyDelete", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphTopologyGet", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphTopologyList", "1.0"),
-                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphTopologySet", "1.0"),
+                new Tuple<string, string, string>("DirectMethodsforLiveVideoAnalyticsonIoTEdge", "GraphInstanceActivate", "1.0.4"),
```

Comment:
While 1.0.4 will be the new version of the module, API versions are not assigned a patch number. The API version will be 1.0.

Reply:
yes,, this is a autogenerated code.. its not being used for api version in the direct method calls. This is more of a Info.